### PR TITLE
DBZ-3809 Support images push from Jenkins

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.redhat.com/browse/DBZ-3809

Requires https://github.com/debezium/docker-images/pull/237

### CODE REVIEW

File: jenkins-jobs/job-dsl/deploy_docker_images.groovy [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    properties {
+        githubProjectUrl('https://github.com/debezium/docker-images')
+    }
+
+    logRotator {
+        daysToKeep(7)
+    }
+
+    triggers {
+        cron('@midnight')
```
[the remark this answers is not in the document]
Comment (jpechane) [reply jenkins-jobs/job-dsl/deploy_docker_images.groovy:14]: From the docs
>  @midnight actually means some time between 12:00 AM and 2:59 AM.

[the remark this answers is not in the document]
Comment (gunnarmorling) [reply jenkins-jobs/job-dsl/deploy_docker_images.groovy:14]: LOL, seriously? Thanks, TIL :)

**Comment by gunnarmorling** [comment]
@jpechane, does this take care of the debezium-ui image, too?

**Comment by jpechane** [comment]
@gunnarmorling https://github.com/debezium/docker-images/pull/237/files#diff-30500ce3de7e0cf213a95a0bb2ebaf9bfe01b20eead32d3d2ef796beb5d256e2R76

**Comment by gunnarmorling** [comment]
Ok, applied. Can you trigger the jobs now so we see whether things work? Or do we have to wait for the first nightly run?

Ah, btw. one more question on that "midnight" expression: is it ensured still that the container image build picks up _today's_ nightly libraries instead of those from the day before? I.e. does the nightly lib build run before midnight?

**Comment by jpechane** [comment]
@gunnarmorling It is already tested, the jobs were deployed and build using my branch,
Nightly image is built right after each build snapshot job - https://github.com/debezium/debezium/blob/master/jenkins-jobs/job-dsl/release_debezium_nightly_image.groovy#L27 which is run 4 times a day
And regarding midnight - TBH this concept is purely fictional and it does not make sense to try to sync something on it as midnight is different in Europe and USA and also depends on where Jenkins is located.

**Comment by gunnarmorling** [comment]
Ok, so IIUC, the image would contain libraries which are 6h old at most?

**Comment by jpechane** [comment]
@gunnarmorling Yes

**Pull request merged**